config/settings.yml

```diff
@@ -43,7 +43,7 @@ email:
   from: 'noreply@example.com'
   fake:
     enabled: false
-    to:
+    to: nucore@tablexi.com
```

Comment:
I might recommend formatting this as a list. I think that'll make the downstream merge conflicts just _a little_ easier.

```yaml
to:
  - nucore@tablexi.com
```